models/repo.go

```diff
@@ -265,31 +265,63 @@ func (repo *Repository) innerAPIFormat(e Engine, mode AccessMode, isParent bool)
 			parent = repo.BaseRepo.innerAPIFormat(e, mode, true)
 		}
 	}
+	hasIssues := false
+	if _, err := repo.GetUnit(UnitTypeIssues); err == nil {
```

Comment:
You should use `repo.getUnit(e, UnitTypeIssues)` to instead `repo.GetUnit`.

Reply:
Fixed! Thanks for giving me the fix for that!